tests/test_cleaneval.py

```diff
@@ -0,0 +1,486 @@
+    extractor_config.rm_footer_lists = False
+    return clean_metahtml(raw_html, extractor_config)
+
+def clean_metahtml_5(raw_html):
+    extractor_config = metahtml.content.ExtractorConfig()
+    extractor_config.rm_header_lists = False
+    return clean_metahtml(raw_html, extractor_config)
+
+def clean_metahtml(raw_html, extractor_config=metahtml.content.ExtractorConfig()):
+    '''
+    Arguments:
+        raw_html        String  Raw downloaded html input
+    Return:
+        cleaned_html    String  Cleaned html by metahtml library
+    >>> clean_metahtml('<text id="http://medicine.ucsf.edu/housestaff/handbook/HospH2002_C7.htm" title="HEMATOLOGY / ONCOLOGY" encoding="windows-1252"> <html> <head> <meta name=Generator content="Microsoft Office HTML Filter 2.0"> <meta http-equiv=Content-Type content="text/html; charset=windows-1252"> <meta name=Originator content="Microsoft Word 9"> <title>HEMATOLOGY / ONCOLOGY</title> <style> <!-- p.MsoNormal, li.MsoNormal, div.MsoNormal { margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:"Times New Roman";}')
+    ''
+    '''
+    url = __get_url(raw_html)
+    res = metahtml.parse(raw_html, url, None, False, extractor_config, True)
+    if res['content'] is not None:
+        cleaned_html = res['content']['best']['value']['text']
+    else:
+        cleaned_html = ''
+    return cleaned_html
+
+def clean_trafilatura_no_fallback(raw_html):
+    '''                                                          
+    Arguments:
+        raw_html        String  Raw downloaded html input
+    Return:
+        cleaned_html    String  Cleaned html by trafilatura library only
+    Faster than with fallback option but less content extracted
+    >>> clean_trafilatura_no_fallback('<div><h3>Test Header</h3><p>Content 123</p></div>')
+    'Content 123'
+    '''
+    url = __get_url(raw_html)
+    cleaned_html = trafilatura.extract(raw_html, url, no_fallback=True)
+    if cleaned_html is None:
+        cleaned_html = ''
+    return cleaned_html
+
+def clean_newspaper3k(raw_html):
+    '''
+    Arguments:
+        raw_html        String  Raw downloaded html input
+    Return:
+        cleaned_html    String  Cleaned html by newspaper3k library
+    '''
+    parser = parsers.Parser()
+    cleaned_html = parser.fromstring(raw_html)
+    return cleaned_html.text_content()
+
+def clean_trafilatura_with_fallback(raw_html):
+    '''
+    Arguments:
+        raw_html        String  Raw downloaded html input
+    Return:
+        cleaned_html    String  Cleaned html by trafilatura library with fallback by justtext and readability-lxml
+    Slower but more content extracted. We opt the doctest for this function as there is a required minimum file size
+    '''
+    url = __get_url(raw_html)
+    cleaned_html = trafilatura.extract(raw_html, url)
+    if cleaned_html is None:
+        cleaned_html = ''
+    return cleaned_html
+
+# Rename function names
+clean_cleaneval.__name__ = 'cleaneval'
```

Comment:
this is really ugly; I believe you're trying to do this to make the output tables print nicer, but there are much better ways to get this done; you basically never want to modify the `__name__` variable as this can mess up code that assumes `__name__` is set correctly